### PR TITLE
Adding cariddi gif and doc .tape

### LINKED ISSUE #4: Update GIF for cariddi
https://github.com/edoardottt/cariddi

Use [`charmbracelet/vhs`](https://github.com/charmbracelet/vhs) to customize the GIF. 

- The GIF must reproduce the exact same commands you can see on the GIF now
- Be sure to add here both the .tape and the GIF file.

Let me know if you need some guidance :)

### PR DESCRIPTION
**Pull request by PaolaDaft**
Issue #4 is done :)

### CODE REVIEW

File: cariddi/cariddi.tape [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# Display:
+#   Hide                            Hide the subsequent commands from the output
+#   Show                            Show the subsequent commands in the output
+
+Output cariddi.gif
+
+Require echo
+
+Set Shell "bash"
+Set FontSize 32
```
Comment (edoardottt) [inline cariddi/cariddi.tape]: same as scilla let's reduce the font size

File: cariddi/cariddi.tape [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+Set Width 1200
+Set Height 600
+
+Type "echo http://testphp.vulmweb.com/ | cariddi" Sleep 500ms  Enter
+
+Sleep 5s
```
Comment (edoardottt) [inline cariddi/cariddi.tape]: same here, sleep longer like 8s

File: cariddi/cariddi.tape [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+Output cariddi.gif
+
+Require echo
+
+Set Shell "bash"
+Set FontSize 32
+Set Width 1200
+Set Height 600
+
+Type "echo http://testphp.vulmweb.com/ | cariddi" Sleep 500ms  Enter
```
Comment (edoardottt) [inline cariddi/cariddi.tape]: here we should sleep a lil bit longer so the user can read the command typed, like 1500ms

**Review by edoardottt — CHANGES_REQUESTED** [review]
I've added some changes to do...

anyway... the output is not the same as before. I tried on my machine and if I use
```console
echo http://testphp.vulnweb.com/ | cariddi
```

I get the same result as the old gif... maybe you tried many times and you are limited?

EDIT: it's `vulnweb`, not `vulmweb`

**Comment by PaolaDaft** [comment]
I've already made all the changes you requested  c;

**Review by edoardottt — APPROVED** [review]
lgtm

**Pull request merged**